### ISSUE TITLE
False positive with `ConstructorInvokesOverridable` (2)

### ISSUE THREAD

**Issue by Stephan202** [google/error-prone#659]
Consider this class with a copy constructor:
```java
public class Dummy {
    private final String property;

    Dummy(Dummy other) {
        this.property = other.getProperty();
    }

    public String getProperty() {
        return this.property;
    }
}
```

Running with `-Xep:ConstructorInvokesOverridable:ERROR` yields:
```
Dummy.java:5: error: [ConstructorInvokesOverridable] Constructors should not invoke overridable methods.
        this.property = other.getProperty();
                                         ^
    (see http://errorprone.info/bugpattern/ConstructorInvokesOverridable)
1 error
```

In this case `#getProperty()` may indeed have been overridden by a subclass, but we may expect `other` to be fully constructed at this point. (If not, its reference leaked somewhere else, and _that_ location should be flagged.)

**Comment by cushon** [comment]
98d25b520237aacb941bdf27ad4b73ffddacbb14
